client/webserver/site/src/js/register.js

```diff
@@ -162,9 +162,11 @@ export default class RegistrationPage extends BasePage {
 
       return
     }
-    this.fee = res.fee
+    this.fee = res.exch.feeAsset.amount
 
-    page.feeDisplay.textContent = Doc.formatCoinValue(res.fee / 1e8)
+    page.feeDisplay.textContent = Doc.formatCoinValue(this.fee / 1e8)
+    const dcrAsset = res.exch.assets['42'] // res.exch.markets['dcr_btc']
```

Comment:
Can the comment go?

Could create a reverse mapping of `BipIDs` in `doc.js` for so we could do something like `const dcrAsset = res.exch.assets[BipIDSymbols['dcr']]` for readability, but this is going to change soon anyway, so this is fine too.